driver/src/main/java/org/neo4j/driver/v1/Values.java

```diff
@@ -27,6 +27,7 @@
 import java.util.Map;
 
 import org.neo4j.driver.v1.exceptions.ClientException;
+import org.neo4j.driver.v1.internal.util.Extract;
```

Comment:
This is a circular dependency from the API to the impl - which means we could not swap out the implementation of the API. We don't need to fix this here, but I'd like to resolve it after this PR goes in. I've got a branch with a build plugin that fails the build if we have dependencies from the API to impl.